<launch>
  <param name="min_x" value="0.3" />
  <param name="min_y" value="-0.7" />
  <param name="min_z" value="0.15" />
  <param name="max_x" value="1.1" />
  <param name="max_y" value="0.7" />
  <param name="max_z" value="1.7" />
  <param name="fitness_threshold" value="0.0075" />
  <param name="leaf_size" value="0.01" />
  <param name="sigma_threshold" value="8" />
  <param name="nms_radius" value="0.02" />
  <param name="num_threads" value="8" />
  <param name="max_samples" value="1000" />
  <param name="num_candidates" value="1000" />
  <param name="is_tabletop" value="false" />
  <param name="orientation_tolerance" value="0.0254" />
  <param name="orientation_tolerance" value="0.15" />
</launch>
